std/math.d

```diff
@@ -5054,6 +5054,33 @@ real remquo(real x, real y, out int n) @trusted nothrow @nogc  /// ditto
         assert(0, "remquo not implemented");
 }
 
+///
+@safe @nogc nothrow unittest
+{
+    version(Posix)
+    {
+        assert(remainder(5.1, 3.0).feqrel(-0.9) > 16);
+        assert(remainder(-5.1, 3.0).feqrel(0.9) > 16);
+        assert(remainder(0.0, 3.0) == 0.0);
+
+        assert(remainder(1.0, 0.0) is -real.nan);
+        assert(remainder(-1.0, 0.0) is -real.nan);
```

Comment:
These two tests assert for me on Android/ARM with the latest ldc 1.10 beta, as Bionic's `remainderl` returns `real.nan` instead. @kinke, same on linux/armhf?

@JackStouffer, where did you get the values from the table above, wouldn't seem to matter what the sign is, as the result is undefined?